test/src/080-tracing/main

```diff
@@ -0,0 +1,61 @@
+cvmfs_test_name="tracing"
+
+cvmfs_run_test() {
+    logfile=$1
+    sudo sh -c "echo \"export GLITE_VERSION='util'\" > /etc/cvmfs/config.d/grid.cern.ch.local"
+    repo="grid.cern.ch"
+    echo "Setup..."
+    rm -f /tmp/cvmfs-$repo.trace.log 2> /dev/null
```

Comment:
In addition, I think it would be good to set up a signal handler later on that removes the file when the function exists.  This can be done with `trap`.  You can check test case 011 for a blue print.